src/App.tsx

```diff
@@ -21,9 +38,22 @@ const App = () => {
   const [drawerOpen, setDrawerOpen] = useState(isGreaterThanLgBreakpoint());
 
   useEffect(() => {
-    const isAccessTokenPresent = getAuthSession('accessToken') !== null;
-    const isTokenExpired = checkAuthStatusService();
-    setAuthenticated(isTokenExpired || isAccessTokenPresent);
+    const checkAuth = async () => {
+      const isAccessTokenPresent = getAuthSession('accessToken') !== null;
+      let isSessionAlive = false;
+      if(!isAccessTokenPresent){
+        // New user
+        isSessionAlive = false;
+      }
+      else{
+        // existing user
```

Comment:
We should also check if the token time is expired or not. Only do a renew call if the token time is expired.

Reply:
@mdshamoon is it okay now?